_src/static/sass/components/_buttons.scss

```diff
@@ -0,0 +1,389 @@
+    color: palette(grayscale-cool, base);
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, mid-dark);
+        border-color: palette(primary, mid-dark);
+        color: palette(grayscale, white);
+    }
+
+    // STATE: is pressed or active
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        border-color: palette(primary, dark);
+    }
+
+    // STATE: is disabled
+    &:disabled,
+    &.is-disabled {
+        background: $transparent;
+        border-color: palette(grayscale-cool, x-light);
+        color: palette(grayscale-cool, mid-light);
+    }
+}
+
+
+// ----------------------------
+// #PRIMARY
+// ----------------------------
+.btn-primary {
+    background: palette(primary, mid-dark);
+    border-color: palette(primary, mid-dark);
+    color: palette(grayscale, white);
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, base);
+        border-color: palette(primary, base);
+    }
+
+    // STATE: is pressed or active
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, mid-dark);
+        border-color: palette(primary, mid-dark);
+    }
+
+    // STATE: is disabled
+    &:disabled,
+    &.is-disabled {
+        background: palette(grayscale, x-light);
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, base);
+    }
+}
+
+
+// ----------------------------
+// #SECONDARY
+// ----------------------------
+.btn-secondary {
+    background: $transparent;
+    border-color: $transparent;
+    color: palette(primary, base);
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        border-color: palette(grayscale-cool, trans);
+    }
+
+    // STATE: is pressed or active
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        border-color: palette(primary, dark);
+        color: palette(primary, mid-dark);
+    }
+
+    // STATE: is disabled
+    &:disabled,
+    &.is-disabled {
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, mid-light);
+    }
+}
+
+
+// ----------------------------
+// #SECONDARY-ALT
+// ----------------------------
+.btn-secondary-alt {
+    background: $transparent;
+    border-color: $transparent;
+    color: palette(grayscale-cool, base);
+
+    // STATE: hover and focus
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(grayscale-cool, base);
+        border-color: palette(grayscale-cool, base);
+        color: palette(grayscale, white);
+    }
+
+    // STATE: is pressed or active
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        border-color: palette(primary, dark);
+    }
+
+    // STATE: is disabled
+    &:disabled,
```

Comment:
Probably not. Let me review and make tweaks as needed. Might impact our scale/palettes. That okay?